simulator-ui/src/main/resources/static/src/app/components/scenarios/scenario.html

```diff
@@ -1,13 +1,16 @@
 <h1 class="page-header"><i class="fa fa-cubes"></i> Scenarios</h1>
 
+<hr/>
```

Comment:
this is not the same as `<br/>` - is it intentional?

Reply:
[Horizontal line](https://developer.mozilla.org/de/docs/Web/HTML/Element/hr)

Reply:
I know and since you know it too, I think it's intentional 😅 that's fine.